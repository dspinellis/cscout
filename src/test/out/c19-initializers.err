Processing workspace TestWS
Processing directory test/c
Entering directory test/c
Processing project Prj1
Processing file c19-initializers.c
/cscout/src/test/c/c19-initializers.c:20: warning: too many initializers
The text leading to this warning is: [p2 = &(struct point){.y = 14, 18}]
Done processing file c19-initializers.c
Done processing project Prj1
Processing project Prj2
Processing file c19-initializers.c
/cscout/src/test/c/c19-initializers.c:20: warning: too many initializers
The text leading to this warning is: [p2 = &(struct point){.y = 14, 18}]
Done processing file c19-initializers.c
Processing file prj2.c
Done processing file prj2.c
Done processing project Prj2
Exiting directory test/c
Done processing directory test/c
Done processing workspace TestWS
Processing identifiers
3%6%9%12%15%18%21%24%27%30%33%36%39%42%45%48%51%54%57%60%63%66%69%72%75%78%81%84%87%90%93%96%100%
