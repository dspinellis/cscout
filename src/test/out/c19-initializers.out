Table: Ids
EID	NAME	READONLY	UNDEFMACRO	MACRO	FUNMACRO	MACROARG	CPPCONST	CPPSTRVAL	DEFCCONSTVAL	NOTDEFCCONSTVAL	EXPCCONSTVAL	NOTEXPCCONSTVAL	ORDINARY	SUETAG	SUMEMBER	LABEL	TYPEDEF	ENUM	YACC	FUN	CSCOPE	LSCOPE	UNUSED
49	main	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	0
91	MAXDIGIT	0	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
321	x	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
541	qqq	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	1	1
759	empty2	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
821	foo	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	0	1	1
879	i	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	1
1079	k	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0
2489	i	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	1
2975	__DATE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3169	ES	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	1	0	0	0	0	0	0
3249	k2	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0
3295	__TIME__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3575	__FILE__	1	0	1	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	1
3865	__LINE__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
4055	__STDC__	1	0	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	1
4599	point	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
4719	x	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
4749	y	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
4799	p2	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0
4839	p	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0
5355	_cscout_dummy1	1	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	1	1	0	0
6989	color	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
7099	r	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
7129	g	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
7159	b	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
7299	rectangle	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0
7569	tl	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
7609	br	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
7799	c	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0
7859	r	0	0	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0
Table: Tokens
FID	FOFFSET	EID
2	297	2975
2	329	3295
2	357	3575
2	386	3865
2	405	4055
2	471	49
2	535	5355
2	558	5355
4	4	49
4	75	759
4	87	879
4	100	759
4	107	1079
4	140	1079
4	152	759
4	248	2489
4	316	3169
4	321	3169
4	324	3249
4	359	3249
4	365	3169
4	459	4599
4	471	4719
4	474	4749
4	479	4799
4	483	4839
4	489	4749
4	497	4719
4	506	4839
4	518	4599
4	526	4719
4	534	4749
4	543	4799
4	557	4599
4	565	4719
4	574	4749
4	584	4799
4	598	4599
4	606	4749
4	698	6989
4	709	7099
4	712	7129
4	715	7159
4	729	7299
4	750	4599
4	756	7569
4	760	7609
4	773	6989
4	779	7799
4	785	7859
4	794	7609
4	802	4749
4	810	4719
4	822	7799
4	829	7099
4	837	7159
4	845	7129
4	857	7569
4	865	4719
4	873	4749
4	888	7859
4	900	7299
4	915	7609
4	923	4749
4	931	4719
4	943	7799
4	950	7099
4	958	7159
4	966	7129
4	978	7569
4	986	4719
4	994	4749
4	1007	7859
4	1019	7299
4	1036	4749
4	1044	4719
4	1058	4719
4	1066	4749
4	1080	7099
4	1088	7159
4	1096	7129
4	1111	7859
4	1123	7299
4	1138	7609
4	1151	4599
4	1160	4749
4	1168	4719
4	1180	7799
4	1192	6989
4	1201	7099
4	1209	7159
4	1217	7129
4	1229	7569
4	1242	4599
4	1251	4719
4	1259	4749
4	1273	7859
4	1285	7299
4	1300	7569
4	1308	4719
4	1316	4749
4	1328	7609
4	1341	4599
4	1350	4749
4	1358	4719
4	1372	7099
4	1380	7159
4	1388	7129
5	8	91
5	31	321
5	41	49
5	53	541
5	81	821
Table: Rest
FID	FOFFSET	CODE
2	287	\u0000a\u0000a#define 
2	305	  
2	320	\u0000a#define 
2	337	 
2	348	\u0000a#define 
2	365	 
2	377	\u0000a#define 
2	394	 1\u0000a#define 
2	413	 1\u0000a\u0000a
2	466	\u0000aint 
2	475	();\u0000a
2	522	\u0000astatic void 
2	549	(void) { 
2	572	(); }\u0000a
3	152	\u0000a\u0000a\u0000a#pragma includepath 
3	195	\u0000a
3	239	\u0000astatic void _cscout_dummy2(void) { _cscout_dummy2(); }\u0000a
4	0	int\u0000a
4	8	()\u0000a{\u0000a	
4	66	\u0000a	struct 
4	81	 {int 
4	88	;};\u0000a	struct 
4	106	 
4	108	 = {};	
4	138	\u0000a	
4	141	 = (struct 
4	158	){};	
4	191	\u0000a\u0000a	
4	243		int 
4	249	 = ((const int[]){ 64, 64, 64, 63 })[2];\u0000a\u0000a	typedef struct empty {} 
4	318	;\u0000a	
4	323	 
4	326	 = {};		
4	357	\u0000a	
4	361	 = (
4	367	){};		
4	401	\u0000a\u0000a	
4	450	\u0000a	struct 
4	464	 { int 
4	472	, 
4	475	;} *
4	481	, 
4	484	 = {.
4	490	 = 1, .
4	498	 = 2};\u0000a	
4	507	 = (struct 
4	523	){.
4	527	 = 4, .
4	535	 = 8};\u0000a	
4	545	 = &(struct 
4	562	){.
4	566	 = 14, .
4	575	 = 18};\u0000a	
4	586	 = &(struct 
4	603	){.
4	607	 = 14, 18};	
4	640	\u0000a\u0000a	
4	689	\u0000a	struct 
4	703	 {int 
4	710	, 
4	713	, 
4	716	; };\u0000a	struct 
4	738	 {\u0000a		struct 
4	755	 
4	758	, 
4	762	;\u0000a		struct 
4	778	 
4	780	;\u0000a	} 
4	786	 = {\u0000a		.
4	796	 = { .
4	803	 = 4, .
4	811	 = 8 },\u0000a		.
4	823	 = { .
4	830	 = 4, .
4	838	 = 1, .
4	846	 = 12},\u0000a		.
4	859	 = { .
4	866	 = 1, .
4	874	 = 2 },\u0000a	};\u0000a\u0000a	
4	889	 = (struct 
4	909	){\u0000a		.
4	917	 = { .
4	924	 = 4, .
4	932	 = 8 },\u0000a		.
4	944	 = { .
4	951	 = 4, .
4	959	 = 1, .
4	967	 = 12},\u0000a		.
4	980	 = { .
4	987	 = 1, .
4	995	 = 2 }\u0000a	};\u0000a	
4	1008	 = (struct 
4	1028	){\u0000a		{ .
4	1037	 = 4, .
4	1045	 = 8 },\u0000a		{ .
4	1059	 = 1, .
4	1067	 = 2 },\u0000a		{ .
4	1081	 = 4, .
4	1089	 = 1, .
4	1097	 = 12},\u0000a	};\u0000a\u0000a	
4	1112	 = (struct 
4	1132	){\u0000a		.
4	1140	 = (struct 
4	1156	){ .
4	1161	 = 4, .
4	1169	 = 8 },\u0000a		.
4	1181	 = (struct 
4	1197	){ .
4	1202	 = 4, .
4	1210	 = 1, .
4	1218	 = 12},\u0000a		.
4	1231	 = (struct 
4	1247	){ .
4	1252	 = 1, .
4	1260	 = 2 }\u0000a	};\u0000a\u0000a	
4	1274	 = (struct 
4	1294	){\u0000a		.
4	1302	 = { .
4	1309	 = 1, .
4	1317	 = 2 },\u0000a		.
4	1330	 = (struct 
4	1346	){ .
4	1351	 = 4, .
4	1359	 = 8 },\u0000a		{ .
4	1373	 = 4, .
4	1381	 = 1, .
4	1389	 = 12},\u0000a	};\u0000a}\u0000a
5	0	#define 
5	16	 11\u0000aextern int 
5	32	;\u0000aextern 
5	45	();\u0000aint 
5	56	;\u0000a\u0000a
5	80	\u0000a
5	84	() {\u0000a	
5	107	}\u0000a\u0000a
Table: Projects
PID	NAME
23	unspecified
24	Prj1
25	Prj2
Table: IdProj
EID	PID
49	24
759	24
879	24
1079	24
2489	24
2975	24
3169	24
3249	24
3295	24
3575	24
3865	24
4055	24
4599	24
4719	24
4749	24
4799	24
4839	24
5355	24
6989	24
7099	24
7129	24
7159	24
7299	24
7569	24
7609	24
7799	24
7859	24
49	25
91	25
321	25
541	25
759	25
821	25
879	25
1079	25
2489	25
2975	25
3169	25
3249	25
3295	25
3575	25
3865	25
4055	25
4599	25
4719	25
4749	25
4799	25
4839	25
5355	25
6989	25
7099	25
7129	25
7159	25
7299	25
7569	25
7609	25
7799	25
7859	25
Table: Files
FID	NAME	RO
2	host-defs.h	1
3	host-incs.h	1
4	c19-initializers.c	0
5	prj2.c	0
Table: Filemetrics
FID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NCOPIES	NINCFILE	NPFUNCTION	NFFUNCTION	NPVAR	NFVAR	NAGGREGATE	NAMEMBER	NENUM	NEMEMBER
2	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	16	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	2	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	1	0	0	0	0	0	0
2	1	548	367	29	0	3	22	61	47	NULL	1	1	0	0	0	5	0	0	5	37	2	0	0	2	0	3	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	2	5	8	1	1	5	7	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
3	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	15	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	NULL	NULL	0	0	0	0	0	0	0	0
3	1	243	187	13	0	2	13	48	28	NULL	1	1	0	0	0	1	0	0	0	16	1	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
4	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	110	0	3	2	NULL	NULL	NULL	NULL	NULL	NULL	NULL	474	22	123	0	48	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	2	0	NULL	38	1	0	NULL	8	0	NULL	NULL	NULL	1	0	0	0	5	9	0	0
4	1	1364	281	355	1	8	55	53	116	NULL	3	2	0	0	0	0	0	0	0	473	22	123	0	48	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	0	19	1	0	0	8	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
5	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	7	0	1	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	18	3	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	NULL	4	4	0	NULL	4	0	NULL	NULL	NULL	1	0	2	0	0	0	0	0
5	1	95	32	17	2	0	11	20	13	NULL	1	1	0	0	0	1	0	0	1	21	3	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	4	0	1	5	4	0	1	5	0	0	1	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	NULL
Table: FileProj
FID	PID
2	24
3	24
4	24
1	25
2	25
3	25
4	25
5	25
Table: LineProj
FID	PID	LNUM
2	24	1
2	24	2
2	24	3
2	24	4
2	24	5
2	24	6
2	24	7
2	24	8
2	24	9
2	24	10
2	24	11
2	24	12
2	24	13
2	24	14
2	24	15
2	24	16
2	24	17
2	24	18
2	24	19
2	24	20
2	24	21
2	24	22
3	24	1
3	24	2
3	24	3
3	24	4
3	24	5
3	24	6
3	24	7
3	24	8
3	24	9
3	24	10
3	24	11
3	24	12
3	24	13
4	24	1
4	24	2
4	24	3
4	24	4
4	24	5
4	24	6
4	24	7
4	24	8
4	24	9
4	24	10
4	24	11
4	24	12
4	24	13
4	24	14
4	24	15
4	24	16
4	24	17
4	24	18
4	24	19
4	24	20
4	24	21
4	24	22
4	24	23
4	24	24
4	24	25
4	24	26
4	24	27
4	24	28
4	24	29
4	24	30
4	24	31
4	24	32
4	24	33
4	24	34
4	24	35
4	24	36
4	24	37
4	24	38
4	24	39
4	24	40
4	24	41
4	24	42
4	24	43
4	24	44
4	24	45
4	24	46
4	24	47
4	24	48
4	24	49
4	24	50
4	24	51
4	24	52
4	24	53
4	24	54
4	24	55
2	25	1
2	25	2
2	25	3
2	25	4
2	25	5
2	25	6
2	25	7
2	25	8
2	25	9
2	25	10
2	25	11
2	25	12
2	25	13
2	25	14
2	25	15
2	25	16
2	25	17
2	25	18
2	25	19
2	25	20
2	25	21
2	25	22
3	25	1
3	25	2
3	25	3
3	25	4
3	25	5
3	25	6
3	25	7
3	25	8
3	25	9
3	25	10
3	25	11
3	25	12
3	25	13
4	25	1
4	25	2
4	25	3
4	25	4
4	25	5
4	25	6
4	25	7
4	25	8
4	25	9
4	25	10
4	25	11
4	25	12
4	25	13
4	25	14
4	25	15
4	25	16
4	25	17
4	25	18
4	25	19
4	25	20
4	25	21
4	25	22
4	25	23
4	25	24
4	25	25
4	25	26
4	25	27
4	25	28
4	25	29
4	25	30
4	25	31
4	25	32
4	25	33
4	25	34
4	25	35
4	25	36
4	25	37
4	25	38
4	25	39
4	25	40
4	25	41
4	25	42
4	25	43
4	25	44
4	25	45
4	25	46
4	25	47
4	25	48
4	25	49
4	25	50
4	25	51
4	25	52
4	25	53
4	25	54
4	25	55
5	25	1
5	25	2
5	25	3
5	25	4
5	25	5
5	25	6
5	25	7
5	25	8
5	25	9
5	25	10
5	25	11
Table: LinePos
FID	FOFFSET	LNUM
2	0	1
2	4	2
2	51	3
2	54	4
2	100	5
2	119	6
2	122	7
2	184	8
2	231	9
2	234	10
2	284	11
2	288	12
2	289	13
2	321	14
2	349	15
2	378	16
2	397	17
2	416	18
2	417	19
2	467	20
2	479	21
2	523	22
3	0	1
3	4	2
3	51	3
3	54	4
3	78	5
3	97	6
3	100	7
3	149	8
3	153	9
3	154	10
3	155	11
3	196	12
3	240	13
4	0	1
4	4	2
4	11	3
4	13	4
4	67	5
4	92	6
4	139	7
4	192	8
4	193	9
4	243	10
4	290	11
4	291	12
4	320	13
4	358	14
4	402	15
4	403	16
4	451	17
4	505	18
4	542	19
4	583	20
4	641	21
4	642	22
4	690	23
4	721	24
4	741	25
4	764	26
4	782	27
4	791	28
4	819	29
4	854	30
4	882	31
4	886	32
4	887	33
4	912	34
4	940	35
4	975	36
4	1002	37
4	1006	38
4	1031	39
4	1053	40
4	1075	41
4	1105	42
4	1109	43
4	1110	44
4	1135	45
4	1177	46
4	1226	47
4	1267	48
4	1271	49
4	1272	50
4	1297	51
4	1325	52
4	1367	53
4	1397	54
4	1401	55
5	0	1
5	20	2
5	34	3
5	49	4
5	58	5
5	59	6
5	80	7
5	81	8
5	89	9
5	107	10
5	109	11
Table: Definers
PID	CUID	BASEFILEID	DEFINERID
24	4	4	2
25	4	4	2
25	5	5	2
Table: Includers
PID	CUID	BASEFILEID	INCLUDERID
24	2	2	1
24	4	3	1
24	4	4	1
25	2	2	1
25	2	2	1
25	4	3	1
25	4	4	1
25	5	3	1
25	5	5	1
Table: Providers
PID	CUID	PROVIDERID
24	2	2
24	4	4
25	2	2
25	2	2
25	4	4
25	5	5
Table: IncTriggers
PID	CUID	BASEFILEID	DEFINERID	FOFFSET	LEN
24	4	4	2	471	4
25	4	4	2	471	4
25	5	5	2	471	4
Table: Functions
ID	NAME	ISMACRO	DEFINED	DECLARED	FILESCOPED	FID	FOFFSET	FANIN
49	main	0	1	1	0	4	4	0
821	foo	0	1	1	0	5	81	0
5355	_cscout_dummy1	0	1	1	1	2	535	1
Table: FunctionDefs
FUNCTIONID	FIDBEGIN	FOFFSETBEGIN	FIDEND	FOFFSETEND
49	4	12	4	1402
821	5	88	5	108
5355	2	557	2	577
Table: FunctionMetrics
FUNCTIONID	PRECPP	NCHAR	NCCOMMENT	NSPACE	NLCOMMENT	NBCOMMENT	NLINE	MAXLINELEN	MAXSTMTLEN	MAXSTMTNEST	MAXBRACENEST	MAXBRACKNEST	BRACENEST	BRACKNEST	NULINE	NPPDIRECTIVE	NPPCOND	NPPFMACRO	NPPOMACRO	NTOKEN	NSTMT	NOP	NUOP	NNCONST	NCLIT	NSTRING	NPPCONCATOP	NPPSTRINGOP	NIF	NELSE	NSWITCH	NCASE	NDEFAULT	NBREAK	NFOR	NWHILE	NDO	NCONTINUE	NGOTO	NRETURN	NASM	NTYPEOF	NPID	NFID	NMID	NID	NUPID	NUFID	NUMID	NUID	NLABEL	NMACROEXPANDTOKEN	NGNSOC	NMPARAM	NFPARAM	NEPARAM	FANIN	FANOUT	CCYCL1	CCYCL2	CCYCL3	CSTRUC	CHAL	IFLOW
49	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	110	0	2	2	NULL	NULL	NULL	NULL	NULL	NULL	NULL	468	22	123	4	48	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	18	0	0	NULL	7	0	NULL	9	NULL	0	NULL	0	0	1	1	1	0.0	1111.82	0.0
49	1	1353	281	353	1	8	53	53	116	NULL	2	2	0	0	0	0	0	0	0	468	22	123	4	48	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	18	0	0	0	7	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	1111.82	0.0
821	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	0	0	0	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	1	0	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	0	0	NULL	0	0	NULL	16	NULL	0	NULL	0	0	1	1	1	0.0	0.0	0.0
821	1	21	14	3	1	0	3	17	0	NULL	0	0	0	0	0	0	0	0	0	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	NULL	0	NULL	0	0	0	1	1	1	0.0	0.0	0.0
5355	0	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	0	0	1	NULL	NULL	NULL	NULL	NULL	NULL	NULL	5	1	0	0	0	0	0	NULL	NULL	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	NULL	1	0	1	NULL	1	0	NULL	8	NULL	0	NULL	1	1	1	1	1	1.0	0.0	1.0
5355	1	22	0	3	0	0	1	21	5	NULL	0	1	0	0	0	0	0	0	0	5	1	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	0	1	0	1	0	1	0	1	0	0	NULL	0	NULL	0	1	1	1	1	1	1.0	0.0	1.0
Table: FunctionId
FUNCTIONID	ORDINAL	FID	FOFFSET
49	0	2	471
821	0	5	81
5355	0	2	535
Table: Fcalls
SOURCEID	DESTID
5355	5355
Done
